VisualStudio.gitignore

```diff
@@ -183,6 +183,7 @@ AppPackages/
 BundleArtifacts/
 Package.StoreAssociation.xml
 _pkginfo.txt
+.appx
```

Comment:
@julienshepherd cool, so what's different about your setup that means it's not creating it under `AppPackages/`? If the earlier rule isn't something we can rely on then perhaps the other rule needs to go away...